### PR TITLE
Replace HTML in includes folder

### PR DESCRIPTION
**Pull request by gewarren**
Contributes to #25479. Inspired by https://dev.azure.com/ceapex/Engineering/_workitems/edit/467273.

### CODE REVIEW

File: includes/migration-guide/runtime/asp/no-longer-able-set-enableviewstatemac-false.md [hunk 1]
```diff
@@ -2,17 +2,17 @@
...
 #### Suggestion
 
 EnableViewStateMac must be assumed to be true, and any resulting MAC errors must be resolved (as explained in [this guidance](https://support.microsoft.com/kb/2915218), which contains multiple resolutions depending on the specifics of what is causing MAC errors).
 
-| Name    | Value       |
-|:--------|:------------|
-| Scope   |Major|
-|Version|4.5.2|
-|Type|Runtime|
+| Name    | Value   |
```
Comment (IEvangelist) [inline includes/migration-guide/runtime/asp/no-longer-able-set-enableviewstatemac-false.md:11]: I hope that you used the [Evenly distribute selected table](https://review.docs.microsoft.com/en-us/help/contribute/docs-authoring/reformat-table?branch=master#evenly-distribute-selected-table) option from the DAP?!

**Review by IEvangelist — APPROVED** [review]
Looks good to me, :shipit: when you're ready.

File: includes/migration-guide/runtime/asp/no-longer-able-set-enableviewstatemac-false.md (hunk 1 above)
Comment (gewarren) [reply includes/migration-guide/runtime/asp/no-longer-able-set-enableviewstatemac-false.md:11]: I thought I was using your command, but now it looks like it's from the Markdown Table extension.

image

**Comment by gewarren** [comment]
> Looks good to me, :shipit: when you're ready.

Thanks for reviewing!

**Pull request merged**